src/utils/Mulberry32.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This file should not be here.

You can remove it to `test/xxx`.

Reply:
How about creating a `utils` folder under the `test/`?

Reply:
Good idea. :)